import_export_extensions/resources.py

```diff
@@ -194,6 +207,8 @@ def export(
         """Init task state before exporting."""
         if queryset is None:
             queryset = self.get_queryset()
+
+        queryset = self.filter_export(queryset, **kwargs)
```

Comment:
Why did you add it?

Reply:
If you define `filter_export` in resource class and add any filter, `total_objects_count` will be calculated incorrectly because `filter_export` is called inside parent export method
https://github.com/django-import-export/django-import-export/blob/main/import_export/resources.py#L1098

I added a comment